Automattic/wp-calypso
# RTL: fix all ignored rtl ignore directives

#### Changes proposed in this Pull Request
In #28876, #28654, #28655 and a few others, I "fixed" RTL issues by adding `/*rtl:ignore*/` comments to the sass files. 
This worked nicely in test mode, where we load the `-debug` css files.

However, those comments are removed by node-sass, and by the time rtlcss is run, they're not there anymore.

Using "loud" comments prevents this issue.
(All other changes are a result of some auto optimization)

#### Testing instructions
Set your environment to `production`: `CALYPSO_ENV=production npm start`
and follow the test intructions in #28876, #28654, #28655 
Alternatively: check that the `rtl:ignore` directives are still present in `style.css`.

Fixes #28873
